### ISSUE TITLE
Icon not showing on tray with HAVE_APP_INDICATOR

### ISSUE THREAD

**Issue by xaengceilbiths** [osdlyrics/osdlyrics#59]
The self-compiled binary on archlinux has a bug(icon not showing on tray).
I followed the PKGBUILD to build the binary and the bug appears. But the official binary works just well. I checked ol_trayicon.c and found the difference(The official one don't have this opt-dependency).
My desktop environment is gnome-flashback. I hope users of other Linux can confirm this bug.

**Comment by xaengceilbiths** [comment]
According to https://github.com/osdlyrics/osdlyrics/issues/59 maybe this issue is related to PKGBUILD?
https://git.archlinux.org/svntogit/community.git/tree/repos/community-x86_64?h=packages/osdlyrics

**Comment by xaengceilbiths** [comment]
I tested budgie and xfce4.
This only happens on gnome-flashback.

**Comment by Corax26** [comment]
Sorry for the massive delay and never implementing #35 despite my promise :/ When configuring, what do you get "AppIndicator: yes"? In which case, could you configure with `--enable-appindicator=no` and see if that works?

**Comment by allencch** [comment]
Using Arch Linux as well. I added these packages, and they solve the status icon problem,

```
python2-future 
python2-gobject 
python2-libappindicator
```

**Comment by PedroHLC** [comment]
Yep, with time those got split, but it would be better to use python 3 instead.
If someone could take ownership and just add these new deps would be good.
(Also I should delete `osdlyrics-pedrohlc` and maybe read as `osdlyrics49` instead)
